pkg/scorecard_result.go

```diff
@@ -80,6 +81,36 @@ func (r *ScorecardResult) AsCSV(showDetails bool, logLevel zapcore.Level,
 	return nil
 }
 
+// AggregatedScore returns an aggregate score out of checker.MaxResultScore.
+func (r *ScorecardResult) AggregatedScore(checkDocs docs.Doc) (int, error) {
+	// TODO: calculate the score and make it a field
+	// of ScorecardResult
+	weights := map[string]int{"Critical": 7, "High": 5, "Medium": 3, "Low": 1}
```

Comment:
Should we try equal distribution
critical: 10
high: 7.5
medium: 5
low: 2.5
easier to explain in our terms of 1-10 scale ?

Reply:
yes main thing i want is aggregate score to show in 1-10 range.

Reply:
sure, that's the case.